charts/icm-as/templates/_infrastructureMonitoring.tpl

```diff
@@ -0,0 +1,87 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Creates init-containers
+*/}}
+{{- define "icm-as.infrastructureMonitoringContainer" -}}
+{{- if .Values.infrastructureMonitoring.enabled }}
+- name: infrastructure-monitoring
+  image: {{ .Values.infrastructureMonitoring.image.repository }}
+  imagePullPolicy: {{ .Values.infrastructureMonitoring.image.pullPolicy | default "IfNotPresent" }}
+  env:
+  {{- if .Values.mssql.enabled }}
+  - name: QUARKUS_DATASOURCE_JDBC_URL
```

Comment:
The name should start with something like INFRA_MONIT_DATAS... instead of QUARKUS_.

Reply:
That is not possible because it configures the default data source provided by Quarkus (name is defined by Quarkus).